### ISSUE TITLE
Is there a tutorial for taint analysis?

### ISSUE THREAD

**Issue by Radon10043** [secure-software-engineering/phasar#481]
Hi, I want to use phasar for taint analysis, but I can't find a tutorial of using phasar for taint analysis in wiki or homepage, can anyone tell me how to use it for taint analysis? Thank you very much!

**Comment by MMory** [comment]
I agree there is a lack of documentation here.

**Comment by fabianbs96** [comment]
Yes, unfortunately, we didn't have time for creating a documentation regarding this yet.

**Comment by fabianbs96** [comment]
https://github.com/secure-software-engineering/phasar/wiki/FAQ#how-to-use-phasar-for-taint-analysis
